scripts/coverage.sh

```diff
@@ -22,80 +22,80 @@ fi
 
 mkdir -p build-coverage
 cd build-coverage
-platform='unknown'
+
+path_to_compiler=''
+
 unamestr=`uname`
-if [[ "$unamestr" == 'Linux' ]]; then
-   # Use GCC for Linux
-   cmake -DCMAKE_CXX_COMPILER_LAUNCHER=$launcher -DCMAKE_BUILD_TYPE=Debug -DCMAKE_C_COMPILER=gcc -DCMAKE_CXX_COMPILER=g++ -DENABLE_COVERAGE=ON ..
-elif [[ "$unamestr" == 'Darwin' ]]; then
-   # Use Clang for OS X
-   cmake -DCMAKE_CXX_COMPILER_LAUNCHER=$launcher -DCMAKE_BUILD_TYPE=Debug -DCMAKE_C_COMPILER=/usr/local/opt/llvm/bin/clang -DCMAKE_CXX_COMPILER=/usr/local/opt/llvm/bin/clang++ -DENABLE_COVERAGE=ON ..
+if [[ "$unamestr" == 'Darwin' ]]; then
+   # Use homebrew clang for OS X
+   path_to_compiler='/usr/local/opt/llvm/bin/'
 fi
 
+cmake -DCMAKE_CXX_COMPILER_LAUNCHER=$launcher -DCMAKE_BUILD_TYPE=Debug -DCMAKE_C_COMPILER=$path_to_compiler'clang' -DCMAKE_CXX_COMPILER=$path_to_compiler'clang++' -DENABLE_COVERAGE=ON ..
+
 cores=$(grep -c ^processor /proc/cpuinfo 2>/dev/null || sysctl -n hw.ncpu)
 make hyriseTest -j $((cores / 2))
 cd -
 
 rm -fr coverage; mkdir coverage
 ./build-coverage/hyriseTest build-coverage --gtest_filter=-SQLiteTestRunnerInstances/*
 
-if [[ "$unamestr" == 'Darwin' ]]; then
-  # LLVM has its own way of dealing with coverage...
-  # https://llvm.org/docs/CoverageMappingFormat.html
+# LLVM has its own way of dealing with coverage...
+# https://llvm.org/docs/CoverageMappingFormat.html
   
-  # merge the profile data using the llvm-profdata tool:
-  /usr/local/opt/llvm/bin/llvm-profdata merge -o ./default.profdata ./default.profraw
+# merge the profile data using the llvm-profdata tool:
+$path_to_compiler'llvm-profdata' merge -o ./default.profdata ./default.profraw
```

Comment:
i think this is easier to read `"${path_to_compiler}llvm-profdata"`. not that is especially nice to read but i was really confused about the single quotes